composer.json

```diff
@@ -46,6 +46,7 @@
         "drupal/entity_reference_display": "1.2.0",
         "drupal/entity_reference_revisions": "1.6",
         "drupal/environment_indicator": "3.5",
+        "drupal/events_log_track": "^1.1",
```

Comment:
Could you please pin the version to a specific version and not a minimum version requirement?

The caret would indicate version `1.10` would be used in the event it would become available, where an equality declaration would prevent unwanted/unexpected change.